packages/schema/src/config/app.ts

```diff
@@ -274,14 +274,7 @@ export default defineUntypedSchema({
      * Customize Nuxt Nuxt SpaLoader element attributes.
      * @type {typeof import('@unhead/schema').HtmlAttributes}
      */
-    spaLoaderAttrs: {
-      $resolve: async (val: undefined | null | Record<string, unknown>, get) => {
-        const spaLoaderId = await get('app.spaLoaderId')
-        return defu(val, {
-          id: spaLoaderId === false ? undefined : (spaLoaderId || '__nuxt-spa-loader'),
```

Comment:
@danielroe We need to ensure that the element's ID is set. These changes break the preloader.

Reply:
ahh, of course. I was just intending to get rid of the reference to the nonexistent spaPreloaderId.

Reply:
... and what do you mean it breaks the preloader?